.github/scripts/automergePR.js

```diff
@@ -0,0 +1,89 @@
+// This script uses JSDoc annotations to get TypeScript checks
+// Why aren't we using TypeScript directly here?
+// `actions/github-script` doesn't support TypeScript scripts directly :/
+
+/**
+ * Note: we can remove typeof below once we upgrade TypeScript
+ * otherwise we get a crash, see https://github.com/microsoft/TypeScript/issues/36830
+ * @typedef {ReturnType<typeof import('@actions/github').getOctokit>} GitHub
+ * @typedef {import('@actions/github').context} Context
+ */
+
+const AUTOMERGE_LABEL = 'automerge'
+
+/**
+ * @param {Object} obj - An object.
+ * @param {GitHub} obj.github
+ * @param {Context} obj.context
+ * @param {Array.<string>} obj.allowedUpdatedFiles
```

Comment:
Curious, why the `.`?

Reply:
i followed the jsdoc guide https://jsdoc.app/tags-type.html without thinking much about it, but digging further, it seems like [the dot isn't needed](https://stackoverflow.com/a/65601923)??? how annoying, i'll remove it